jobs/mongodb_restore_job.go

```diff
@@ -0,0 +1,149 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jobs
+
+import (
+	"context"
+	"net/url"
+	"os/exec"
+	"regexp"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// This regexp matches backup entity name.
+var lastBackupR = regexp.MustCompile(`^Backup snapshots:\n(  (\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z).*)`)
```

Comment:
either `lastBackupRE` or `lastBackupRegexp` would be better

Reply:
Changed to `lastBackupRE`